pegasus/sites.v3/code.org/public/teacher-dashboard/section_manage.haml

```diff
@@ -71,82 +71,6 @@ content-type: text/ng-template
   %div.jumbotron{'ng-show' => "section.id && section.students.length <= 0 && section.login_type === 'google_classroom'", style: "line-height: 36px;"}
     = I18n.t('dashboard_students_new_google_classroom_section')
 
-  %table{style: "width:950px; table-layout:fixed;", 'ng-show' => 'section.students.length > 0', "ng-form" => "allForm"}
-    %colgroup
-      %col{width: "300"}/
-      %col{width: "100"}/
-      %col{width: "100"}/
-      %col{width: "200"}/
-      %col{width: "250"}/
-    %tr
-      %th.manage-th
-        %a{href: "", "ng-click" => "order = 'name'"}= I18n.t('dashboard_students_name')
```

Comment:
Likewise any strings in here? (Not sure which are still used elsewhere, there are a lot).